ci/scripts/deploy_develop.sh

```diff
@@ -68,8 +68,22 @@ if [[ $GITHUB_COMMIT_MESSAGE != *"ci(release): generate CHANGELOG.md for version
     yarn gh-pages --dist coverage --dest development/coverage -m "ci(docs): generate coverage with xcov for version ${GIT_TAG}"
 
     echo "Generate test report"
+    # remove default styles
+    sed -i -e 's/header { position: fixed;width: 100%;background: rgba(249, 254, 255, 0.9);margin: 0;padding: 10px;}//g' ./fastlane/test_output/report.html
```

Comment:
Hi, @hectorerb 
We should add these styles in the css files of the project site.
And this will be a task for the Web Developers.